shipyard_blueprints/googledrive/shipyard_googledrive/googledrive.py

```diff
@@ -1,13 +1,15 @@
+import os
 import json
 import tempfile
-import os
+
 from googleapiclient.discovery import build
 from googleapiclient.http import MediaFileUpload
 from google.oauth2 import service_account
 from shipyard_templates import CloudStorage, ExitCodeException
 from typing import Optional, Dict, List, Any, Union
 from googleapiclient.http import MediaIoBaseDownload
-from shipyard_googledrive import drive_utils
+
+from shipyard_blueprints.googledrive.shipyard_googledrive import drive_utils
```

Comment:
I don't think we want that full path import (`from shipyard_blueprints.googledrive.shipyard_googledrive import ...` ) in place. That may work if you're running it from the root, but is a stand alone package that is not run from the root. So I were to do a version change, build, and then publish this, we would receive an import error not recognizing `shipyard_blueprints`.